hydrolib/core/dflowfm/polyfile/parser.py

```diff
@@ -458,7 +458,7 @@ def _is_empty_line(line: str) -> bool:
     @staticmethod
     def _is_name(line: str) -> bool:
         stripped = line.strip()
-        return len(stripped) >= 1 and line[0] != "*" and " " not in stripped
+        return len(stripped) >= 1 and line[0] != "*"
```

Comment:
should the documentation of read_polyfile be updated?
It stated:  `- Name consisting of a non-blank character string`

Reply:
Done